classes/security/Validation.php

```diff
@@ -195,21 +195,13 @@ public static function checkCredentials($username, $password)
      *
      * @return bool
      */
-    public static function isAuthorized($roleId, $contextId = 0)
+    public static function isAuthorized($roleId, ?int $contextId = Application::SITE_CONTEXT_ID)
     {
         if (!self::isLoggedIn()) {
             return false;
         }
 
-        if ($contextId === -1) {
```

Comment:
I take it this case was no longer in any calling code?

Reply:
Yep, I didn't find anything using it.